src/EditorFeatures/TestUtilities/ChangeSignature/AbstractChangeSignatureTests.cs

```diff
@@ -62,7 +62,8 @@ protected override ParseOptions GetScriptOptions()
             string expectedErrorText = null,
             int? totalParameters = null,
             bool verifyNoDiagnostics = false,
-            ParseOptions parseOptions = null)
+            ParseOptions parseOptions = null,
+            int expectedSelectedIndex = -1)
```

Comment:
💡 Might be better to default this to `0` so it's always covered by testing.

Reply:
I tried, but in a number of tests the `ParameterConfiguration` is null. 
I don't want to make the assertion conditional on null, as that could hide issues, so I think the current design is safer.

---
In reply to: [260739080](https://github.com/dotnet/roslyn/pull/33453#discussion_r260739080) [](ancestors = 260739080)